packages/expo-updates/README.md

```diff
@@ -350,6 +350,7 @@ import * as Updates from 'expo-updates';
 ### Constants
 
 - **`Updates.manifest` (_object_)** - If `expo-updates` is enabled, this is the [manifest](https://docs.expo.io/versions/latest/workflow/how-expo-works/#expo-development-server) object for the update that's currently running. In development mode, or any other environment in which `expo-updates` is disabled, this object is empty.
+- **`Updates.releaseChannel` (_string_)** - The name of the release channel currently configured in this standalone or bare app.
```

Comment:
When we include expo-updates in the managed dev client, won't we expose the release channel through this same field?

```suggestion
- **`Updates.releaseChannel` (_string_)** - The name of the release channel currently configured in this standalone or bare app. In development clients, this is the name of the release channel of the currently running update.
```